### PR TITLE
STR-54 -- FE: Create the admin page with the ability to take in an excel sheet and pass it to the backend

### PR DESCRIPTION
**Pull request by ekno27**
# Description
Allows admin to upload an excel sheet that parses through the file and will send the sheet's contents to the backend.
  
# Testing
1. Upload a file that isnt excel (word, mp3, etc), you should be notified to use an excel file
2. upload the excel file that was sent by me on the iSTART slack channel 
3. to check the parsed file, check the admin component using vue devtools
4. run jest to make sure all tests pass
5. enjoy a cool drink in this pleasant weather
## NOTE II, THE SEQUEL: styling is dependent on design's mockup, so it was not included as part of this task

# Installation
Sheets.js's xlsx was used in order to parse through the file. To install, just ``yarn install``

### CODE REVIEW

**Review by BrianLinggadjaja — DISMISSED** [review]
Everything else is good including test, I just noticed that some extensions don't put out a message such as ***`(.pdf, .exe, etc.)`***

File: resources/js/views/admin.vue [hunk 1]
```diff
@@ -1,13 +1,88 @@
...
+  name: 'admin',
+  mixins: [changeRouteTitle],
+  data() {
+    return {
+      participants: [],
+      incorrectFileType: false
+    }
+  },
+  methods: {
+    handleFileChange(evt) {
```
Comment (frlzjosh) [inline resources/js/views/admin.vue]: change `evt` to something like `fileEvent`

File: tests/jsUnit/views/admin.spec.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+describe('Admin.vue', () => {
+  
+  
+  const $route = {
+    meta: {
+      title: 'ISTART'
+    } 
+  }
+  const wrapper = shallowMount(Admin, {
+    mocks: {$route}
```
Comment (frlzjosh) [inline tests/jsUnit/views/admin.spec.js:13]: nice!

**Review by frlzjosh — CHANGES_REQUESTED** [review]
very small changes needed.

**Review by frlzjosh — APPROVED** [review]
wow edgar! L0okS gR3At

**Review by andrewMETALAB — CHANGES_REQUESTED** [review]
In your instructions, you said: "to check the parsed file, check the admin component using vue devtools".
But I don't see any difference in the vue dev tools after uploading your excel file and clicking "submit", so I'm a little confused.
Come show me what I should be seeing next time you're in? Thank you sir.

**Review by andrewMETALAB — APPROVED** [review]

**Pull request merged**